setup.cfg

```diff
@@ -14,3 +14,4 @@ exclude = venv,build,tutorial,.tox,.asv
 [mypy]
 disallow_untyped_defs = True
 ignore_missing_imports = True
+exclude = venv|build|docs|tutorial|optuna/storages/_rdb/alembic
```

Comment:
`exclude` was introduced in 0.812.
http://mypy-lang.blogspot.com/2021/02/mypy-0812-released.html

Reply:
Do you think we should add the version constraint for mypy, e.g., `mypy>=0.812`?

Reply:
I think it's OK to leave as is IMO (no additional version constraint).